tests/application/cms/test_page_service.py

```diff
@@ -428,6 +428,25 @@ def test_create_new_major_version_does_not_duplicate_uploads(self):
         assert len(measure_version.uploads) == 1
         assert len(new_version.uploads) == 0
 
+    @pytest.mark.parametrize("version_type", [t for t in NewVersionType])
+    def test_create_measure_version_creates_associations_rather_than_copies_of_existing_data_sources(
+        self, version_type
+    ):
+        measure_version = MeasureVersionFactory(version="1.0")
```

Comment:
I'd be tempted to explicitly create a data source and assign `data_sources=[data_source]` here, rather than relying on the factory? Possibly even assign two?